doc/api/errors.md

```diff
@@ -450,18 +450,35 @@ added properties.
 
 #### error.code
 
-Returns a string representing the error code, which is always `E` followed by
+Is a string representing the error code, which is always `E` followed by
 a sequence of capital letters, and may be referenced in `man 2 intro`.
 
 #### error.errno
 
-Returns a number corresponding to the **negated** error code, which may be
+Is a number corresponding to the **negated** error code, which may be
 referenced in `man 2 intro`. For example, an `ENOENT` error has an `errno` of
 `-2` because the error code for `ENOENT` is `2`.
 
+In some modules, this property is the same string as `error.code`.
+
 #### error.syscall
 
-Returns a string describing the [syscall][] that failed.
+Is a string describing the [syscall][] that failed.
+
+#### error.path
+
+Is a string of the specified invalid pathname in case of using some modules
+like `fs` or `child_process`.
+
+#### error.address
+
+Is a string describing the address that is not available in case of using some
```

Comment:
This appears only in the error that comes from net and dgram. I think that is why it was missed in the doc. I told him to add what modules it happens in. How about this?
````
When present (e.g. in net and dgram), it is a string describing the address that is not available. 
````